_includes/pbc/all/install-features/202304.0/install-the-inventory-management-feature.md

```diff
@@ -0,0 +1,687 @@
+
+
+This document describes how to ingrate the [Inventory Management](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html) feature into a Spryker project.
+
+{% info_block errorBox %}
+
+The following feature integration guide expects the basic feature to be in place.
+
+The current feature integration guide adds the following functionality:
+
+* [Warehouse Management](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html)
+* [Add to cart from catalog page](/docs/scos/user/features/{{site.version}}/cart-feature-overview/quick-order-from-the-catalog-page-overview.html)
+* [Warehouse address](/docs/pbc/all/warehouse-management-system/{{site.version}}/inventory-management-feature-overview.html#defining-a-warehouse-address)
+
+{% endinfo_block %}
+
+
+## Install feature core
+
+Follow the steps below to install the Inventory Management feature core.
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+
+| NAME         | VERSION          |
+|--------------|------------------|
+| Spryker Core | {{site.version}} |
```

Comment:
In the warehouse allocation module you have a dependency on the sales module.
Please clarify mapping with PO and AA regarding it, I see 2 options:
- Adjust this IG and add dependency to Order Management feature, which looks wrong (IMO)
- Create separate feature - `warehouse-allocation` that would depend on Order and Inventory management features.